data/tlg0732/tlg013/__cts__.xml

```diff
@@ -0,0 +1,9 @@
+<ti:work xmlns:ti="http://chs.harvard.edu/xmlns/cts" groupUrn="urn:cts:greekLit:tlg0732"
+    xml:lang="grc" urn="urn:cts:greekLit:tlg0732.tlg013">
+    <ti:title xml:lang="grc">Ἠθικὰ προβλήματα [Sp.]</ti:title>
```

Comment:
The xml:lang attributes here should technically probably be "mul" since [Sp.] is not Greek.

Reply:
Don't forget to change the xml:lang on the `<ti:title>` element as well.